cedar-policy/src/api/err.rs

```diff
@@ -579,11 +578,53 @@ pub enum ContextJsonError {
     /// Error deserializing the JSON into a Context
     #[error(transparent)]
     #[diagnostic(transparent)]
-    JsonDeserialization(#[from] ContextJsonDeserializationError),
+    JsonDeserialization(#[from] context_json_errors::ContextJsonDeserializationError),
     /// The supplied action doesn't exist in the supplied schema
+    #[error(transparent)]
+    #[diagnostic(transparent)]
+    MissingAction(#[from] context_json_errors::MissingActionError),
+}
+
+impl ContextJsonError {
+    /// Construct a `ContextJsonError::MissingAction`
```

Comment:
Should this be a rustdoc link?

Reply:
Does rustdoc support links like this?

Reply:
`pub(crate)`, so not much of an issue either way